### ISSUE TITLE
HEADS UP: Refactor coming...Prepare.

### ISSUE THREAD

**Issue by NathanWalker** [NathanWalker/angular-seed-advanced#119]
If you are using this seed, please read carefully:
There is a very nice refactor coming down the pipeline soon from the parent seed:
https://github.com/mgechev/angular2-seed/pull/959

It should hopefully be the last fairly major refactor to land, but you should prepare your projects if you intend to merge in latest upstream when it lands.

Preparation checklist:
- make sure your custom code is mostly in your own `project.framework` folder as mentioned here: https://github.com/NathanWalker/angular2-seed-advanced#how-best-to-use-for-your-project
- make sure any custom project config changes you've made are in your `project.config.ts`. 
- If you've made custom tasks, make sure those are in separate files much like the seed base itself is for it's various 'out of the box' tasks.

Doing the above will ensure that you can merge in the upstream changes when the refactor lands with _lesser_ conflicts. You will likely have some but they shouldn't be too bad to sort through.

In the event you have steered way off course and have tons of modifications, getting things together and into their own file or framework would allow you to download a new zip and just drop your custom stuff in.

I will close this once the refactor lands here; it will still be another week or more.

**Comment by vincentpalita** [comment]
Thanks for the heads up @nathanwalker!

**Comment by joshwiens** [comment]
I really wish I could get Minko to run a "develop" like branch to merge this stuff into first instead of just hammering everything into master :(

**Comment by JakePartusch** [comment]
@NathanWalker, #61 will be on hold until these changes come through :/

**Comment by NathanWalker** [comment]
The refactor is not going to happen due to complex issues with SystemJS, so disregard this 👍 
Multi-platforms are supported by this seed now and more can be integrated using similar setups as was done with {N} and electron integration already.

@JakePartusch Feel free to continue with angular-universal. This could be housed in a `server` folder in the root similar to how `nativescript` folder exists in root.
